README.md

```diff
@@ -138,6 +138,12 @@ vbrickEmbed.on('videoLoaded', data => {
 });
 ```
 
+Use the .off method to stop listening for events:
+
+```
+vbrickEmbed.off('videoLoaded');
```

Comment:
This function needs a 2nd argument, which is the listener you added with `.on`

```js
const onVideoLoaded = data => {
		console.log(`Video loaded: ${JSON.stringify(data)}`);
};
vbrickEmbed.on('videoLoaded', onVideoLoaded);

// ....

vbrickEmbed.off('videoLoaded', onVideoLoaded);
```

Reply:
done